attributes/default.rb

```diff
@@ -5,6 +5,9 @@
 default["collectd"]["checksum"]           = "853680936893df00bfc2be58f61ab9181fecb1cf45fc5cddcb7d25da98855f65"
 default["collectd"]["interval"]           = 10
 default["collectd"]["read_threads"]       = 5
+default['collectd']['WriteQueueLimitHigh'] = 2_000_000
```

Comment:
Can you please make these changes consistent with all of the other attributes?

- Single quotes to double quotes
- Snake case names
- `'false'` as boolean

In addition, I think the cookbook should use the same default queue limits that `collectd` uses:

```
WriteQueueLimitHigh 1000000
WriteQueueLimitLow   800000

```